lib/phlex/rails/form_fields.rb

```diff
@@ -0,0 +1,68 @@
+module Phlex
+  module Rails
+    class FormFields < Phlex::HTML
+			def initialize(form:, scopes:)
+				@scopes = Array.wrap(scopes).compact
+				@form = form
+			end
+
+			def self.input_field(method_name, type:)
+				define_method method_name do |field, value: @model.attributes[field.to_s], **attributes|
+					input(
+						name: field_name(field),
+						type: type,
+						value: value,
+						**attributes
+					)
+				end
+			end
+
+			def authenticity_token_field
+				input(
+					name: "authenticity_token",
+					type: "hidden",
+					value: @_view_context.form_authenticity_token
+				)
+			end
+
+			def submit(value)
+				input(
+					name: "commit",
+					type: "submit",
+					value: value
+				)
+			end
+
+			def template(&blk)
+				yield_content &blk
+			end
+
+			def field_name(field_name)
+				fields = [*@scopes, field_name].compact
+				field_string = fields[0].to_s.dup
+				field_string << fields[1..-1].map { |f| "[#{f}]" }.join
+				field_string
+			end
+
+			def fields_for(scope, &blk)
+				render FormFields.new(form: @form, scopes: [*@scopes, *Array.wrap(scope)]) do
+					yield_content &blk
```

Comment:
We can just do `yield` now in v1.0.0

Reply:
@Bumppoman Did something change in v1.0.0 to allow for just `yield`? I'm trying to build some tests and this approach for yielding the `FormFields` back from the `Form` object is no longer working. The `form` is yielded instead.

Reply:
Found the issue and [opened](https://github.com/joeldrapper/phlex/issues/403) it in the phlex repo. Will see if the behavior of yielding a different object is considered acceptable and if so will submit a pull request.